microcosm_eventsource/stores.py

```diff
@@ -34,28 +34,39 @@ def retrieve_most_recent_by_event_type(self, event_type, **kwargs):
             self.model_class.event_type == event_type,
         )
 
-    def upsert_on_parent_id(self, instance):
+    def upsert_index_elements(self):
```

Comment:
I did a source scan and couldn't find any references to `upsert_on_parent_id` in our code, so I think this is safe.